pkg/api/admin/platformworkloadidentityroleset_validatestatic.go

```diff
@@ -74,7 +74,7 @@ func (sv platformWorkloadIdentityRoleSetStaticValidator) validateDelta(new, curr
 	err := immutable.Validate("", new, current)
 	if err != nil {
 		err := err.(*immutable.ValidationError)
-		return api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodePropertyChangeNotAllowed, err.Target, err.Message)
+		return api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodePropertyChangeNotAllowed, err.Target, err.Message) //nolint:govet
```

Comment:
Please try to explain why these had to be ignored.

Reply:
Explained [here](https://github.com/Azure/ARO-RP/pull/3797#discussion_r1739404938)